.github/workflows/auto-updates.yaml

```diff
@@ -131,8 +131,10 @@ jobs:
       - name: Update XS-Vendored-Sources-Rust
         run: |
           set -eu
-        
-          OUTPUT=$((CARGO_VENDOR_DIR=vendor_rust/ /usr/share/cargo/bin/dh-cargo-vendored-sources 2>&1 || true) | grep XS-Vendored-Sources-Rust:)
+
+          export CARGO_VENDOR_DIR=vendor_rust/ 
+
```

Comment:
nitpick: I don’t think we need an empty line there, as it only applied to the command below, so let’s group them. After this, you can merge :)